src/pipx/util.py

```diff
@@ -75,14 +75,9 @@ def get_venv_paths(root: Path) -> Tuple[Path, Path]:
         return bin_path, python_path
 
 
-def get_script_output(interpreter: Path, script: str, *args) -> str:
-    # Make sure that Python writes output in UTF-8
-    env = os.environ.copy()
-    env["PYTHONIOENCODING"] = "utf-8"
-    output = subprocess.run(
-        [str(interpreter), "-c", script, *args], stdout=subprocess.PIPE, env=env
-    ).stdout.decode(encoding="utf-8")
-    return output
+def get_script_output(interpreter: Path, script: str, *args: Union[str, Path]) -> str:
+    proc = run_subprocess([interpreter, "-c", script, *args], capture_stderr=False)
+    return proc.stdout
```

Comment:
This is a sequence of bytes that needs to be decoded isn't it?

Reply:
See the above comment.  `proc.stdout` is text not bytes with `universal_newlines` set in `run_subprocess()`.

Reply:
Ah I missed that we are using `universal_newlines`. Thanks.